core/drivers/gic.c

```diff
@@ -221,9 +227,17 @@ static void gic_it_add(struct gic_data *gd, size_t it)
 	write32(mask, gd->gicd_base + GICD_ICENABLER(idx));
 	/* Make it non-pending */
 	write32(mask, gd->gicd_base + GICD_ICPENDR(idx));
+#if defined(CFG_ARM_GICV3)
+	/* Assign it to group1S */
+	write32(read32(gd->gicd_base + GICD_IGROUPR(idx)) & ~mask,
+			gd->gicd_base + GICD_IGROUPR(idx));
+	write32(read32(gd->gicd_base + GICD_IGROUPMODR(idx)) | mask,
+			gd->gicd_base + GICD_IGROUPMODR(idx));
```

Comment:
My only concern here is TF-A can also configure list of Group1S interrupts based on list provided by the platform. If those interrupts and the ones being configured here do not match then there is a problem.

Reply:
@soby-mathew 
If TF-A have done some mismatched configuration for G1S also (which will be unnecessary after this patch) op-tee will override them.  TF-A does not do anything more than configuring those interrupts. correct me if it has any other implication.

Reply:
Ah, I see that that all the interrupts are being initialized in gic_init(). Ok, I agree the config done by TF-A is not a problem in this case.